vllm/engine/output_processor/single_step.py

```diff
@@ -131,22 +130,22 @@ def _process_sequence_group_outputs(self, seq_group: SequenceGroup,
                 # not be used in the future iterations.
                 parent.status = SequenceStatus.FINISHED_ABORTED
                 seq_group.remove(parent.seq_id)
```

Comment:
I'm not sure you can remove the parent.seq_id here, since else it won't be freed in the free_finished_seq call (before the schedule() call)